조이-안지유/movie/src/components/Movies.jsx

```diff
@@ -0,0 +1,28 @@
+import React, {useState} from 'react';
+import "../styles/movies.css";
+import MovieInfo from "./MovieInfo.jsx";
+
+const Movies = (props) => {
+  const {original_title, overview, poster_path, vote_average} = props.data;
```

Comment:
```suggestion
const Movies = ({ original_title, overview, poster_path, vote_average }) => {
```
처음부터 props가 아닌 구조분해할당을 사용해주면 6번 라인 생략 가능해져요